src/htdocs/index.php

```diff
@@ -1,10 +1,107 @@
 <?php
 
 if (!isset($TEMPLATE)) {
-  $TITLE = "GeoServe";
+  include_once '../conf/config.inc.php';
+
+  $TITLE = "Nearby Places";
   include_once 'template.inc.php';
 }
 
+$latitude = param('latitude', null);
+$longitude = param('longitude', null);
+$limit = param('limit', 5);
+if (!$limit) {
```

Comment:
This handles the case when the form field is cleared, the library sets a default limit.  Maybe we should remove the default.